lib/src/views/Clock/ClockNumbers.tsx

```diff
@@ -7,7 +7,7 @@ interface GetHourNumbersOptions {
   ampm: boolean;
   utils: IUtils<MaterialUiPickersDate>;
   date: MaterialUiPickersDate;
-  onChange: (value: number, isFinish?: boolean) => void;
+  onChange: (value: number, isFinish?: boolean | symbol) => void;
```

Comment:
What's `symbol` is for?

```suggestion
  onChange: (value: number, isFinish?: boolean) => void;
```

Reply:
Will be not needed by #1923